opendcs-rest-api/src/main/java/org/opendcs/odcsapi/util/ApiBasicClient.java

```diff
@@ -143,30 +145,27 @@ public void disconnect( )
 				if (input != null)
 					input.close();
 			}
-			catch (Exception e)
+			catch (IOException ex)
 			{
-				System.out.println("Error closing input: "+e.getMessage());
-				e.printStackTrace();
+				throw new IOException("Error closing input: ", ex);
 			}
-			try
+            try
 			{
-				if (output != null)
-					output.close();
-			}
-			catch (Exception e)
+                if (output != null)
+                    output.close();
+            }
+			catch (IOException ex)
```

Comment:
I think we still want to close the socket even if the output.close() fails

Reply:
I did update this in the new commit, the new code is much more readable, but technically if one close fails, the others won't be run.

Readability wise, it might make sense to do it this way?